airflow-core/src/airflow/ui/src/i18n/locales/de/README.md

```diff
@@ -101,6 +111,9 @@ Für die Deutsche Übersetzung wurden die folgenden Terme wie folgt übersetzt
   starten. Von allen Optionen der am ehesten passende Begriff auch wenn es eine
   direkte Nutzung des Begriffs "Triggern" im Deutschen gibt. In der Nutzung im
   UI ist der Begriff "Anstoßen" inhaltlich passender.
+- `Trigger Rule` --> `Auslöse-Regel`: Im Ablauf eines Dag geht es für den
+  Begriff um den Punkt und die Regel zu dem ein abhängiger Task gestartet
```

Comment:
```suggestion
  Begriff um den Zeitpunk und die Regel zu dem ein abhängiger Task gestartet
```

Ich denke was auch möglich wäre ist:

"Im Ablauf eines Dags bestimmt die Auslöse-Regel (in Kombination mit der Position) jedes Tasks unter welchen Bedingungen und zu welchem Zeitpunkt im Dag Lauf dieser Task gestartet werden kann."